packages/lab/src/__tests__/__e2e__/combo-box-next/ComboBoxNext.cy.tsx

```diff
@@ -3,122 +3,439 @@ import * as comboBoxNextStories from "@stories/combo-box-next/combo-box-next.sto
 
 import { CustomFloatingComponentProvider, FLOATING_TEST_ID } from "../common";
 
-const { Default, CustomRenderer, Controlled } =
-  composeStories(comboBoxNextStories);
+const {
+  Default,
+  Readonly,
+  WithDefaultSelected,
+  Disabled,
+  DisabledOption,
+  MultiSelect,
+  WithFormField,
+  Grouped,
+  EmptyMessage,
+  ComplexOption,
+} = composeStories(comboBoxNextStories);
 
-describe("GIVEN a Combobox", () => {
-  it("SHOULD render all its items", () => {
+describe("Given a ComboBox", () => {
+  it("should be able to filter and select an option with a mouse", () => {
+    const selectionChangeSpy = cy.stub().as("selectionChange");
+    cy.mount(<Default onSelectionChange={selectionChangeSpy} />);
+
+    cy.findByRole("combobox").realClick();
+    cy.findByRole("combobox").should("be.focused");
+
+    cy.realType("Ala");
+
+    cy.findByRole("option", { name: "Alabama" }).realClick();
+
+    cy.findByRole("combobox").should("be.focused");
+    cy.findByRole("combobox").should("have.value", "Alabama");
+    cy.get("@selectionChange").should(
+      "have.been.calledWith",
+      Cypress.sinon.match.any,
+      Cypress.sinon.match.array.deepEquals(["Alabama"])
+    );
+  });
+
+  it("should be able to filter and select an option with a keyboard", () => {
+    const selectionChangeSpy = cy.stub().as("selectionChange");
+    cy.mount(<Default onSelectionChange={selectionChangeSpy} />);
+
+    cy.findByRole("combobox").realClick();
```

Comment:
```suggestion
    cy.realPress("Tab");
```

Reply:
This doesn't really affect the test. The test is for selection with a keyboard not the initial focus